src/Page/Definition.elm

```diff
@@ -75,7 +77,7 @@ view model =
 
 renderQuotes : List Key -> Html Msg
 renderQuotes quoteKeys =
-    div [] (List.map (\quoteKey -> p [] [ text (t quoteKey) ]) quoteKeys)
+    blockquote [] (List.map (\quoteKey -> p [ css quoteStyle ] [ text (t quoteKey) ]) quoteKeys)
```

Comment:
If there is only ever one quote we can strip this down now. The `blockquote` tag should only wrap one quote anyway, right?